### ISSUE TITLE
Controlling the swipe threshold for delete

### ISSUE THREAD

**Issue by OzTK** [h6ah4i/android-advancedrecyclerview#266]
My users are telling me that the swipe action doesn't trigger easily enough the delete action. As an example they show me the Gmail app. It is true that just a small swipe is enough to trigger the complete swipe of the item in this app. Would it be possible to control the sensitivity of the swipe to trigger the action attached (in my case a delete)?

**Comment by h6ah4i** [comment]
Hi. Thanks for the suggestion 👍  I will make it configurable, so please wait for a few days.

NOTE: Current threshold is hard coded on the following line;
https://github.com/h6ah4i/android-advancedrecyclerview/blob/0.9.1/library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/swipeable/RecyclerViewSwipeManager.java#L413

**Comment by h6ah4i** [comment]
Hi. I have introduced the `void setSwipeThresholdDistance(int distanceInPixels)` method to `RecyclerViewSwipeManager` in develop branch code. This improvement will be included in the next release, or you can try it now via JitPack.

```gradle
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.h6ah4i:android-advancedrecyclerview:69dced818128e2a3380de8a69478aa431edbc5ab'
}
```

**Comment by OzTK** [comment]
Hi, Thanks a lot for that very fast improvement :) I saw that threshold constant in the source but was not sure it matched this behavior. I will test it in my app and include it in my next release once it gets in yours :)

**Comment by h6ah4i** [comment]
@OzTK Hi. The `setSwipeThresholdDistance()` method is now available in the v0.9.2. Thanks!
